README.md

```diff
@@ -143,10 +143,10 @@ Moja global thrives on community participation, and we appreciate your contribut
 
 You can reach out to us through the following channels:
 
-- [info@moja.global](mailto:info@moja.global) : Ask questions about moja global relevant to the members and management with a point of focus for new user countries and policy makers.
-- [tsc@moja.global](mailto:tsc@moja.global) : Ask questions about contributing code, helping out with development, resolving issues and building new features with moja global teams.
-- [mentorship@moja.global](mailto:mentorship@moja.global) : Ask questions about participating in the community, sponsoring programs, funding opportunity and community development.
-- [Slack community](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw ): Connect with other developers, scientists, policy makers and other interested people over our Slack community.
+- [info@moja.global](mailto:info@moja.global) : Ask questions about Moja global relevant to the members and management with a point of focus for new user countries and policymakers.
+- [tsc@moja.global](mailto:tsc@moja.global) : Ask questions about contributing code, helping out with development, resolving issues and building new features with Moja global teams.
```

Comment:
On the moja global organization, across all the places; moja global should be spelled as `moja global` only without Capital Case in the word. So, we shall consider writing `moja global` only everywhere. Please make this change.

Reply:
okay will change it for sure, thanks

Reply:
Please review the code now.